src/guide.jl

```diff
@@ -1045,8 +1045,8 @@ function render(guide::Title, theme::Gadfly.Theme,
 
     padding = 2mm
     ctx = compose!(
-        context(minwidth=text_width, minheight=text_height + 2padding),
```

Comment:
I personally like this change. The extra padding on top before this merge didn't feel quite right.